includes/class-sensei-quiz.php

```diff
@@ -1451,6 +1451,18 @@ public static function the_user_status_message( $quiz_id ) {
 
 		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output escaped above.
 		echo $message;
+
+		// show incomplete questions info message if there are incomplete questions and is preview.
+		if ( is_preview() ) {
```

Comment:
I don't think the `is_preview()` will do the job here. Once you click the `Preview` button from the backend, you are redirected to the `lesson` preview and not the `quiz` preview. This code is executed for the quiz only.

I think the solution @dadish is suggesting is great - you could only display the message to users that can edit the lesson.